scripts/build.py

```diff
@@ -46,12 +46,15 @@
 
 EXTENSIONS_DIRNAMES_TO_DIRPATHS = {
     'dev_dir': os.path.join('extensions', ''),
+    'dev_dir_for_js': os.path.join('compiled_js', 'extensions', ''),
```

Comment:
Same thing, this is a bit confusing. Is the distinction actually between unminified_js and minified_js? If so maybe we should use those names instead.

Reply:
This is removed in later commits.